Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -125,3 +134,22 @@ extension Issue {
     return issue
   }
 }
+
+/// A function called by the testing library when a failure occurs.
+///
+/// Whenever a test failure (specifically, a non-known ``Issue``) is recorded,
+/// the testing library calls this function synchronously. This facilitates
+/// interactive debugging of test failures by allowing a symbolic breakpoint to
+/// be added specifying the symbol name of this function
+/// (`SWTFailureBreakpoint`), so that the debugger may pause execution and allow
+/// a user to inspect the process state. This function performs no action of its
+/// own.
+///
+/// This function is not part of the public interface of the testing library,
+/// but it is exported and its symbol name must remain stable.
+@_cdecl("SWTFailureBreakpoint")
```

Comment:
Nitpick: the style we're using for C/C++ symbols is `swt_camelCase()`. We should either make this name match that style (`swt_failureBreakpoint()`) or change our style to be consistent.